### PR TITLE
Remove tweakreg_catalog docs; update tweakreg docs

### PR DESCRIPTION
**Pull request by jdavies-st**
Fix bug in doc build

### CODE REVIEW

File: docs/jwst/tweakreg/description.rst [hunk 1]
```diff
@@ -3,7 +3,10 @@ Description
 ===========
 
 This step creates a catalog of point-like sources whose centroids are
-used to aid in aligning images in the "tweakreg" step.
+then used to aligning images using the Tweakreg algorithm.
```
Comment (larrybradley) [inline docs/jwst/tweakreg/description.rst]: aligning -> align

**Comment by pllim** [comment]
Sorry, the conflict was my bad, I was trying to get RTD to build and did a quick fix on `docs/jwst/tweakreg_catalog/index.rst`.

**Comment by jdavies-st** [comment]
Where is this quick fix @pllim ?

**Comment by pllim** [comment]
548b5b2af2cf1fc6101dea9e43dc0fc1b78bc189 :grimacing:

**Comment by pllim** [comment]
And... aae1dbe4a2efdc7a7569 :grimacing: :grimacing:

**Pull request closed without merging**